cmd/livepeer/livepeer.go

```diff
@@ -513,11 +513,14 @@ func main() {
 			// Set price per pixel base info
 			if *pixelsPerUnit <= 0 {
 				// Can't divide by 0
-				panic(fmt.Errorf("The amount of pixels per unit must be greater than 0, provided %d instead\n", *pixelsPerUnit))
+				panic(fmt.Errorf("-pixelsPerUnit must be > 0, provided %d", *pixelsPerUnit))
```

Comment:
Since new error messages were being added for the pricePerUnit checks below I though this would be a good opportunity to:

- Reference the flag name itself in the error message
- Make the error message a bit more concise